Firestore/Example/Tests/Local/FSTMutationQueueTests.mm

```diff
@@ -186,28 +183,6 @@ - (void)testNextMutationBatchAfterBatchID {
   });
 }
 
-- (void)testNextMutationBatchAfterBatchIDSkipsAcknowledgedBatches {
```

Comment:
This test appears to be no longer relevant.

Reply:
Right. The mutation queue now contains only unacknowledged writes now.